samples/build-netlify/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Build on Netlify</title>
+  <script src="https://unpkg.com/contentful-ui-extensions-sdk@3"></script>
+  <link rel="stylesheet" href="https://unpkg.com/contentful-ui-extensions-sdk@3/dist/cf-extension.css" />
+</head>
+
+<body style="margin: 0;">
+  <div id="root" style="display:none">
+    <button id="triggerBuild" class="cf-btn-primary">Build on Netlify</button>
+    <div id="infoBox" style="margin-top: 10px"></div>
+    <ul id="history" style="margin-top: 10px; padding-left: 15px;"></ul>
+  </div>
+  <script>
+
+    function showExtension(params) {
+      document.getElementById('root').style.display = 'block';
+
+      const POLLING_TIME = 3000;
+      const netlifyBuildHookURL = params.instance.netlifyBuildHookURL;
+      const netlifyAPIToken = params.instance.netlifyAPIToken;
+      const netlifySiteId = params.instance.netlifySiteId;
+
+      function triggerBuild() {
+        return window.fetch(netlifyBuildHookURL, {
+          method: "POST"
+        });
+      }
+
+      var interval = null;
+      var $triggerBuildBtn = document.getElementById('triggerBuild');
+      var $infoBox = document.getElementById('infoBox');
+      var $history = document.getElementById('history');
+
+      function checkStatuses() {
+        if (netlifySiteId && netlifyAPIToken) {
+          window.fetch('https://api.netlify.com/api/v1/sites/' + netlifySiteId + '/builds?access_token=' + netlifyAPIToken)
+            .then(function (response) {
+              return response.json()
+            })
+            .then(function (data) {
+              const markup = data.slice(0, 5).map(function (item) {
+                return [
+                  '<li>',
+                    (new Date(item['created_at'])).toLocaleString(),
+                  ' - ',
+                  item.done ? 'Finished' : 'Building...',
+                  '</li>'
+                ].join('');
+              }).join('');
+              $history.innerHTML = markup;
+            });
+        }
+      }
+
+      function startPollingStatuses() {
+        if (netlifySiteId && netlifyAPIToken) {
+          if (interval) {
+            interval.clear();
+          }
+          checkStatuses();
+          interval = window.setInterval(checkStatuses, POLLING_TIME);
+        }
+      }
+
+      $triggerBuildBtn.addEventListener('click', function () {
+        triggerBuild().then(function (data) {
+          $infoBox.innerHTML = 'The build is triggered!';
+          startPollingStatuses();
+        });
+      });
+
+      checkStatuses();
+    }
+
+    window.contentfulExtension.init(function (extension) {
+      if (extension.parameters.instance.access === 'all' ||
```

Comment:
extract to variables so conditional is more readable: `if (accessibleForEverybody || isAdmin) { /* ... */ }`